www/wip_new_website/content/community.md

```diff
@@ -1,20 +1,22 @@
-# Contribute to Roc
+# Community
 
-## Community Values 
+WIP
+
+## Values 
 
 <!-- 
 Work in Progress - Richard is developing this section
 
 Basically it will say something like we expect you to be kind, even if you think someone is being mean to you, don't assume ill intentions etc
 -->
 
-## Meet the Community
+## Meet
 
 - Group Chat [roc.zulipchat.com](https://roc.zulipchat.com/)
 - Github Project [roc-lang/roc](https://github.com/roc-lang/roc)
 - Meetups & Events
 
-## Ideas & Proposals
+## Proposals
 
 - [Good First Issues](https://github.com/roc-lang/roc/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
```

Comment:
Does the "Good First Issues" link still make sense in this spot?

Reply:
Oops, I didnt mean to remove that.